### PR TITLE
Tests: test_edge_2.py

### PR DESCRIPTION
**Pull request by rohit-rk**
test_count_rows, test_avg_rows_w_default and test_sum_rows_w_domain passes.

### CODE REVIEW

**Comment by rohit-rk** [comment]
The if case for partitions.value was just making the query complicated.

`


          ON e0.d0c0 = CASE 
          WHEN (("a.$string") = ('x'))

              THEN 0

          WHEN (("a.$string") = ('y'))

              THEN 1

          WHEN (("a.$string") = ('z'))

              THEN 2

          ELSE 3

          END`
 "on_clause" of join is responsible for getting all the results from both dimensions.
`        
                
                ON (e0.d0c0 = "a.$string" 

                       OR (e0.d0c0 IS NULL AND "a.$string" IS NULL)) 

                 AND (e1.d1c0 = "b.$string"

                        OR (e1.d1c0 IS NULL AND "b.$string" IS NULL))
`

**Comment by klahnakoski** [comment]
there is no change to the number of tests passing. but, I do not disagree with any of these changes.  So I will merge.

File: tests/test_jx/test_edge_1.py [hunk 1]
```diff
@@ -806,16 +806,17 @@ def test_empty_default_domain_w_groupby(self):
...
                             ]
+                      #          {
+                       #             "dataIndex": 0,
+                        #            "value": "b"
+                         #       },
+                          #      {
+                           #         "dataIndex": 1,
+                            #        "value": "c"
+                             #   }
+
```
Comment (klahnakoski) [inline tests/test_jx/test_edge_1.py]: do not comment, just delete the lines next time

**Comment by rohit-rk** [comment]
Please check the tests again, I see three of them are passing.

**Pull request merged**